main.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import(
+  . "github.com/scorelab/gocloud-v2/gocloud"
+)
+
+
+func main(){
+
+    /*
```

Comment:
Why this entire block is commented? If it is not relevant, remove it from the PR.